tool/tctl/common/collection.go

```diff
@@ -51,7 +51,7 @@ func (r *roleCollection) resources() (res []types.Resource) {
 	return res
 }
 
-func (r *roleCollection) writeText(w io.Writer) error {
+func (r *roleCollection) writeText(_ bool, w io.Writer) error {
```

Comment:
Instead of threading an additional boolean flag into each writeXX method for each resource, can you make `verbose bool` a field of a particular "Collection" object? This way it will only be used by collections that need/support it and we won't have to update all of them to pass additional parameter which is most of the time unused.